- World:
    name: world
    lines:
      - [  0,  0, 14,  0 ]
      - [  0,  9, 14,  9 ]
      - [  0,  0,  0,  9 ]
      - [ 14,  0, 14,  9 ]
      - [  2,  2,  5,  2 ]
      - [  2,  4,  5,  4 ]
      - [  2,  2,  2,  4 ]
      - [  5,  2,  5,  4 ]
      - [  2,  5,  7,  5 ]
      - [  2,  7,  7,  7 ]
      - [  2,  5,  2,  7 ]
      - [  7,  5,  7,  7 ]
      - [  8,  5, 11,  5 ]
      - [  8,  7, 11,  7 ]
      - [  8,  5,  8,  7 ]
      - [ 11,  5, 11,  7 ]

- Plugin:
    name: vasco
    spec: vasco-plugin

- VascoRobot:
    name: vasco
    scan_topic: vasco/scan
    pose: [ 1, 1, 0 ]
    goals:
      - [  6,  1,  0,  0.1, 3.2 ]
      - [  9,  4,  0,  0.1, 3.2 ]
      - [  6,  4,  0,  0.1, 3.2 ]
      - [  1,  5,  0,  0.1, 3.2 ]
      - [  1,  1,  0,  0.1, 3.2 ]
    noisy_scanners: true
#    scanner_noise_min_factor:  0.9
#    scanner_noise_max_factor:  1.1
    scanner_noise_min_offset: -0.01
    scanner_noise_max_offset:  0.01

- View:
    name: ground-truth
    camera:  world
    drawings:
      - world
      - vasco_true_drawing
      - vasco_lidar
    window: [ 0.0,  0.0,  1.0,  1.0 ]
    border:  0
